### DISCUSSION TITLE
Windows development build / gamma test

### DISCUSSION THREAD

**Discussion by char101** [ahrm/sioyek#942] [Ideas]
Hi,

I got motivated to compile sioyek to test the new gamma option (https://github.com/ahrm/sioyek/issues/566). Thank you @ahrm for implementing it.

Since there has been quite a time since the last release of sioyek for Windows, I have uploaded my build here

https://github.com/char101/sioyek/releases/tag/20240126

- Sioyek: development branch
- Compiler: VS 2022 x64
- Qt: 6.6.1 (from https://build-qt.fsu0413.me/6.6-series/6.6.0-for-windows/index.html)

This is a screenshot of the gamma option using Computer Modern font which is notoriously thin. Note that it doesn't seem to work in dark mode.

### gamma=1 (default)
![gamma1](https://github.com/ahrm/sioyek/assets/71255/0784826c-24e2-4b46-89d0-34dd73be4063)
### gamma=2
![gamma2](https://github.com/ahrm/sioyek/assets/71255/13e9f3de-cd0a-4679-a24d-1dd321842509)
### gamma=3
![gamma3](https://github.com/ahrm/sioyek/assets/71255/61b3cca8-7c42-4fae-9423-2b1c80a60eb3)

**Comment by ahrm** [comment]
Thanks, I just want to caution everyone that the development version is under heavy development and I can not guarantee that your data created with this version will be compatible with future sioyek versions.

**Comment by char101** [comment]
New build 20240127 with new gamma commit, mupdf-1.23.9.

https://github.com/char101/sioyek/releases/tag/20240126

gamma=1
![custom1](https://github.com/ahrm/sioyek/assets/71255/79ad6fd3-14ba-4ff2-8d0f-c119a2f9e0fb)

gamma=2
![custom2](https://github.com/ahrm/sioyek/assets/71255/b84b94c7-2583-4008-a7aa-607508e0b6a1)

To create a hotkey to increase/decrease gamma

`prefs_user.conf`
```conf
new_js_command _decrease_gamma decrease_gamma.js
new_js_command _increase_gamma increase_gamma.js
```

`keys_user.conf`
```conf
_increase_gamma <C-g>
_decrease_gamma <C-G>
```

`increase_gamma.js`
```javascript
const gamma = Number(sioyek.get_config_no_dialog('gamma').split(' ')[1]);
sioyek.setconfig_gamma(gamma + 0.1);
```

`decrease_gamma.js`
```javascript
const gamma = Number(sioyek.get_config_no_dialog('gamma').split(' ')[1]);
if (gamma > 0.1) sioyek.setconfig_gamma(gamma - 0.1);
```
